suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -246,15 +263,47 @@ export const getAccountTypeTech = (path: string) => {
     return 'TR_ACCOUNT_TYPE_BIP44_TECH';
 };
 
-export const getAccountTypeDesc = (path: string) => {
+type getAccountTypeDescProps = {
+    path: string;
+    accountType?: string;
+    symbol?: string;
+    network?: string;
+};
+
+export const getAccountTypeDesc = ({ path, accountType, network }: getAccountTypeDescProps) => {
+    switch (accountType?.toLowerCase()) {
+        case 'ledger':
```

Comment:
in the method above, you wrapped this switch into `if (networkType !== 'bitcoin') {` condition, is it ok here? bitcoin has also `legacy` type

Reply:
True, it was supposed to be for the normal type only, for legacy it's ok. Changed